modin/core/execution/dask/common/engine_wrapper.py

```diff
@@ -13,36 +13,46 @@
 
 """Module houses class responsible for execution of remote operations."""
 
+from typing import Tuple
+from distributed import Future
 from distributed.client import default_client
+from modin.utils import Invocable
 
 
 class DaskWrapper:
     """The class responsible for execution of remote operations."""
 
     @classmethod
-    def deploy(cls, func, *args, num_returns=1, pure=None, **kwargs):
+    def deploy(
+        cls,
+        invocable,
+        num_returns=1,
+        pure=True,
```

Comment:
Why is this changed to pure=True?

Reply:
I added a comment about this earlier, but it seems to have disappeared after a rebase. `pure=True` is the default behavior for Dask ([see here](https://docs.dask.org/en/stable/futures.html#distributed.Client.submit)); making this the case in our codebase as well may help avoid potential confusion in the future.